packages/react-on-rails-pro/src/StoreRegistry.ts

```
@@ -13,7 +13,7 @@
  */
 
 import CallbackRegistry from './CallbackRegistry.ts';
-import type { Store, StoreGenerator } from '../types/index.ts';
+import type { Store, StoreGenerator } from 'react-on-rails/types';
```

ESLint

[error] `react-on-rails/types` type import should occur before import of `./CallbackRegistry.ts`

(import/order)